sqlglot/expressions.py

```diff
@@ -3131,6 +3134,14 @@ def to_identifier(alias, quoted=None):
     return identifier
 
 
+def to_table(sql_path, pivots=None):
```

Comment:
I think this is the first time the concept of a "sql_path" has entered this file.

Should this function live here or somewhere else?

Reply:
I think it makes sense here alongside the other function to help construct expressions (like `to_alias` and `to_identity`)